### PR TITLE
[FIX] Update `dask-cuda` versions in dev env ymls

### PR DESCRIPTION
**Pull request by trxcllnt**

### CODE REVIEW

**Comment by trxcllnt** [comment]
rerun tests

**Comment by cjnolet** [comment]
rerun tests

File: conda/environments/raft_dev_cuda11.0.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@ dependencies:
 - rapids-notebook-env=22.02.*
 - rapids-doc-env=22.02.*
 - rmm=22.02.*
-- dask-cuda=22.02.*
+- dask-cuda=22.08.
```
Comment (bdice) [inline conda/environments/raft_dev_cuda11.0.yml:18]: To match other repos' pinnings (and other pinnings in this file), I think this needs a trailing `*`:
```suggestion
- dask-cuda=22.08.*
```

File: conda/environments/raft_dev_cuda11.2.yml [hunk 2]
```diff
@@ -15,7 +15,7 @@ dependencies:
 - rapids-notebook-env=22.02.*
 - rapids-doc-env=22.02.*
 - rmm=22.02.*
-- dask-cuda=22.02.*
+- dask-cuda=22.08.
```
Comment (bdice) [inline conda/environments/raft_dev_cuda11.2.yml:18]:
```suggestion
- dask-cuda=22.08.*
```

File: conda/environments/raft_dev_cuda11.4.yml [hunk 3]
```diff
@@ -15,7 +15,7 @@ dependencies:
 - rapids-notebook-env=22.02.*
 - rapids-doc-env=22.02.*
 - rmm=22.02.*
-- dask-cuda=22.02.*
+- dask-cuda=22.08.
```
Comment (bdice) [inline conda/environments/raft_dev_cuda11.4.yml:18]:
```suggestion
- dask-cuda=22.08.*
```

File: conda/environments/raft_dev_cuda11.5.yml [hunk 4]
```diff
@@ -16,7 +16,7 @@ dependencies:
 - rapids-notebook-env=22.02.*
 - rapids-doc-env=22.02.*
 - rmm=22.02.*
-- dask-cuda=22.02.*
+- dask-cuda=22.08.
```
Comment (bdice) [inline conda/environments/raft_dev_cuda11.5.yml:19]:
```suggestion
- dask-cuda=22.08.*
```

**Comment by bdice** [comment]
FYI, I am seeing conflicts from this in rapids-compose -- it would be nice to get this merged to make the rapids-compose combined conda environment solvable. I know CI issues have been blocking a lot of things.

**Comment by bdice** [comment]
This can be closed because #713 was merged.

**Pull request closed without merging**